src/App.css

```diff
@@ -90,11 +88,38 @@ button {
 }
 
 h2 {
-  color: darkblue;
+  color: #4f4e4e;
   text-align: center;
 }
 
 p {
   color: darkblue;
   text-align: center;
 }
+
+ul {
+  list-style-type: none;
+}
+
+.welcome {
+  color: black;
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav-links {
+  display: flex;
+  list-style: none;
+}
+
+.nav-links li {
+  margin-right: 10px;
+}
+
+.nav-links a {
+  text-decoration: none;
+}
```

Comment:
Nice job so far styling your project😉. Kindly note that we are required to stick to the wireframe in this milestone and your project looks different from the given [wireframes](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md)😉.
<img width="902" alt="image" src="https://github.com/berylcl/math-magicians/assets/64643995/01183cf9-7b66-40d7-83e4-7f4a1c72463a">

what is meant here is that you can style the project to your taste (colors, fonts, background, e.t.c.) but let's try to make sure our project is similar to the wireframe when considering the positioning of the elements and the spacings😉.

for example, from your home page, it may be difficult for your users to see the nav links or even know they are links since they are positioned with other texts and this can lead to a bad user experience🥲.
<img width="990" alt="image" src="https://github.com/berylcl/math-magicians/assets/64643995/9bbbeeea-9b3f-4465-8978-deee85a385ef">

Same with your calculator page, kindly position the nav links and the calculator as shown in the wireframe😉. also, do not forget to add the text on the left of the calculator page that says "let's do some math" you can personalize the text, but stick to the structure of the wireframe😉.
<img width="866" alt="image" src="https://github.com/berylcl/math-magicians/assets/64643995/bcf83dfe-07f2-42b7-8c8d-2de0079cdd6d">

Same with your quote section🥲.
<img width="959" alt="image" src="https://github.com/berylcl/math-magicians/assets/64643995/46857a41-0312-4c9a-bf2b-c5e22be7c380">

Kindly ensure the navbar is positioned at the top of the page and has the same styling and appearance on all pages. ensure the structure, positioning and spacings matches the given wireframe👍.